requirements/prod.txt

```diff
@@ -1,48 +1,36 @@
+ansible_vault>=2.1
 argparse==1.4.0
 base58==2.0.1
 backoff==2.1.0
-certifi==2020.11.8
 cryptography==39.0.0
-idna==2.10
-jinja2==3.0
 fuzzywuzzy==0.18.0
-google-api-python-client==2.7.0
-python-levenshtein==0.12.1
 grpcio==1.42.0
-grpcio-tools==1.42.0
 hypothesis==6.47.4
+idna<3,>=2.5
+jsonschema[format-nongpl]>=4.14.0,<=4.17.0
+langchain>=0.0.132
 loguru==0.6.0
 msgpack==1.0.4
 msgpack-numpy==0.4.7.1
 miniupnpc==2.0.2
 munch==2.5.0
 nest_asyncio==1.5.6
 netaddr==0.8.0
+numpy>=1.24.2
 pandas==1.5.2
-psutil==5.9.4
 password_strength==0.0.3.post2
-protobuf==3.19.5
 prometheus_client==0.14.1
-pycryptodome==3.11.0
-py-sr25519-bindings>=0.1.4,<1
-py-ed25519-bindings>=1.0,<2
-py-bip39-bindings>=0.1.9,<1
+protobuf==3.19.5
+psutil==5.9.4
+python-levenshtein==0.12.1
 pyyaml==6.0
-rich==12.5.1
 retry==0.9.2
+rich==12.5.1
 requests==2.25.0
 scalecodec==1.2.0
-sentencepiece==0.1.97
+substrate-interface==1.5.0
 termcolor==2.1.1
-torch==1.13.1
+torch>=1.13.1
```

Comment:
please, do not specify an unlimited range of versions,
_**Why am i saying this?**_
we should deliver code that is tested and which behavior we know.
with this kind of specifications we can end up with bittensor packages installing versions of torch that we did not tested or that do not work at all at that point.

So, to proceed with this... you may specify an upper bound if you want to provide a range of versions (if that is tested) but, please, do not add an unlimited range of versions because it can lead us to people installing bittensor and not working at all because it may triggers the installation of a version that is not compatible. We should test the versions we integrate in our requirements.